README.md

```diff
@@ -928,10 +928,17 @@ All RabbitMQ resources that are currently supported by the RabbitMQ provider, ar
 
 ### Use with Cloudflare
 
-Example:
+Example using a Cloudflare API Key and corresponding email:
+```
+export CLOUDFLARE_API_KEY=[CLOUDFLARE_API_KEY]
```

Comment:
Exporting those variables is required for them to be passed to subprocesses, at least with bash.